WTWSMS/common/traits/VIET_Immersion_Other.txt

```diff
@@ -45,7 +46,7 @@ griot = { #West African Griot
 
 	customizer = no
 	
-	cannot_inherit = yes
+	cannot_inherit = no
```

Comment:
During the mod's period, or today? Those traditions can evolve, and the author of those hade done extensive research.